redbadger/website-honestly
# Car trawler the second

### ⏫  Changes

#### 🎫  Trello Ticket / Github Issue

https://trello.com/c/fh8EKazM/33-cartrawler-case-study-my-account

#### 🗒  Description

Adds the Car trawler My account case study. Contains a refactor for the our-work slices and a re-arrangement of touched files and directories.

#### 🖼  Screenshots

![screen shot 2018-07-25 at 15 57 25](https://user-images.githubusercontent.com/1148118/43208962-7fc8bcc6-9023-11e8-876d-adadc67e8136.jpg)
![screen shot 2018-07-25 at 15 57 34](https://user-images.githubusercontent.com/1148118/43208963-7fde3600-9023-11e8-98bb-a506c09447c5.jpg)
![screen shot 2018-07-25 at 15 57 42](https://user-images.githubusercontent.com/1148118/43208964-7ff7b44a-9023-11e8-91fd-94b5a4f56a10.jpg)


---

### 💣  Test

#### 💥  Areas of risk

- Our work page 
- Car Trawler My Account case study

#### ☑️  Test plan

- check the copy and styling on the new case study
- check the "what to read next" section at the bottom of case studies displays the correct copy and image (these are chosen at random so might need some page refreshing)
- check that when shared on twitter or facebook, correct image and copy displays (can be done with debuggers available for those)
- check for styling regressions on the our-work page

#### 🏄  Browser testing

##### TIER 1

_Full support. Test all front-end PR's against these._

*WIN10*

*Mac High Sierra*

*iOS: iPhone*
- [x] Last 2 major iOS versions

*iOS: iPad*
- [x] Last 2 major iOS versions

*Android*
- [x] Last 3 major Android versions; check with mixture of devices

##### TIER 2 (Functional support; some visual differences allowed)

_Functional support; some visual differences allowed. Only test these if it's mentioned in the test plan._

*WIN 7*

*Mac Sierra*

*iOS: iPhone*
- [ ] +1 major iOS versions compared to Tier 1

*iOS: iPad*
- [ ] +1 major iOS versions compared to Tier 1

*Ubuntu*
- [ ] Chromium

---

### 🕵️‍♀️ Tracking

A scroll tracker has been added to the page.
